### DISCUSSION TITLE
HPC Linux Configuration

### DISCUSSION THREAD

**Discussion by jsevic** [idaholab/moose#25912] [Q&A General] [answered]
Hello,

A Linux question, I am having an x86 HPC machine configured is there anything specific I should tell the Sys Admin on the Linux flavor or install?

Thx!

John Sevic

**Comment by GiudGiud** [comment]
@milljm will be the best counsel on this

**Reply by milljm** [reply, marked as answer]
Our build farm is using Rocky-8 for most targets, and in the process for eventually transitioning to Rocky-9.

My suggestion is Rocky-9.  Or RedHat Enterprise Linux 9 [RHEL], if that is an option. Rocky is a free clone of RHEL.

**Reply by jsevic** [reply]
OK thank you!
